tests/ltp/patches/fix_mkdir_mkdir02.patch

```diff
@@ -0,0 +1,61 @@
+This change is necessary to run this test case in sgx-lkl
+environment. Test case is modified to work in single
+process environment by removing exit code from child and
```

Comment:
You're not removing an exit code, you're removing a call to `exit()`. What is a "wait code"? I couldn't see that in the patch.

Reply:
@letmaik, The 'tst_reap_children()' function makes the process wait for all of its
children.

Reply:
@hukoyu Could you please help rephrase the description to make it more clear? Thanks!

Reply:
In the original test a child process is used, and parent process waits for child process to finish, and child process calls exit to return to parent process. The test case is modified to work in sgx-lkl single
process environment by removing codes related to exit from child and
wait on parent. FORK_OR_VFORK always returns 0 in sgxlkl environment. 
In addition, application in sgx-lkl environment starts with root user and 
admin permissions lost when user changed from root to non-root user. 
Test case is updated to handle this scenario as well.

Reply:
@shaikshavali1 I reprhased the description. Please fix the formatting and length of lines.

Reply:
@hukoyu,  modified the description as suggested.

Reply:
@letmaik ,
Hi Mike,
Could you please re-run the failed jobs in Azure pipe line?

In Azure pipeline build, below mentioned two test cases were failed before starting the test application in enclave. These failure are not related to my commit. 

languages-java-read_file-(debug)-(run-hw)-(8-ethreads) 
basic-global_vars_test-(debug)-(run-sw)-(8-ethreads)